tests/testthat/_snaps/chat.md

```diff
@@ -4,6 +4,7 @@
       chat
     Output
       <ChatOpenAI>
```

Comment:
Maybe should add total number of messages here?

Reply:
Like `<ChatOpenAI messages=5>`? Something like that?

Reply:
Yeah

Reply:
I added the number of messages _not_ including the system prompt, even though the system prompt is printed right below this line. 🤷‍♂️